### DISCUSSION TITLE
flipping storyboard upside down when using HR

### DISCUSSION THREAD

**Discussion by frarg** [ppy/osu#21655] [Ideas]
this might seem like a strange feature at first, but would it be possible to add a toggle to flip storyboards upside down when using HR? a lot of the time storyboard elements are placed where objects are, and when the map is flipped upside down the storyboard kinda stops making sense..
another option i guess would be to add an optional attribute to some storyboard elements that would make them flip automatically when using HR, that way the entire thing doesnt have to be upside down (however this would only work for new storyboards that use it)
i wouldnt mind working on something like this but i just want to know if theres a chance it would actually be implemented before i spend any time on it

**Comment by ghost** [comment]
I don't like the idea of flipping the whole storyboard upside down. Not only is the background going to be upside-down as well, but any lyrics will be flipped.

I think it's a good idea to "anchor" certain storyboard elements to hitobjects, so that not only does this solve HR flipping things upside down, it works for pretty much any mod which moves hitobjects (e.g. magnetized and random mod)

**Reply by frarg** [reply]
i do agree that anchoring certain elements is a much better solution long term, but since there's no way to retroactively do that for every existing storyboard, that's why i suggested adding the toggle to flip the whole thing upside down. yes i know lyrics and stuff would be upside down too, but (for me at least) it would make it more playable than having it right-side up.
i know it would bother some people, which is why i suggested it as a toggleable feature
